.github/actions/download-build-output/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Download Build Output
+description: Download and restore the `dist` folders from github artifact (check ../upload-build-output)
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/download-artifact@v3
+      if: github.event.release == null
```

Comment:
should not be part of the action (which is not used in case of `release` trigger)

Reply:
removed